public/js/fish.js

```diff
@@ -254,40 +430,34 @@ function updateFishers() {
             $('#f' + j).attr('data-profit-season', profitSeason);
             $('#f' + j).attr('data-name', name);
             $('#f' + j).attr('active-fisher', true);
-
-            j++;
         }
+
+        j++;
```

Comment:
Hmm. Would you recall if I had a bug here, or what is the reason this increment is happening outside of the block it used to be in?